main.go

```diff
@@ -183,7 +183,18 @@ func run(c *cli.Context) error {
 func render(tmpl string, vars map[string]string) (string, error) {
 	fm := template.FuncMap{
 		"split": strings.Split,
-	}
+                "file": func(key string) string {
+                        data, err := ioutil.ReadFile(key)
+                        if err != nil {
+                                return ""
```

Comment:
You should return empty string and the error with some context, both here and below